Rebecas_MD_ex.md

```diff
@@ -1,13 +1,13 @@
 # **Rebeca's Markdown exercise**
 This document contains Rebeca's Markdown exercises. In it you will find the following topics:
-* [Hierarchy of Headings](#hierarchy)
-* [Emphasis](#emphasis)
-* [Blockquotes](#blockquotes)
-* [Lists](#lists)
-* [Code](#code)
-* [Tables](#tables)
-* [Links](#links)
-* [Images](#images)
+* [Hierarchy of Headings](#Hierarchy)
```

Comment:
```suggestion
* [Hierarchy of Headings](#hierarchy-of-headings)
```

Internal Markdown links should match the name of the heading that you're linking to. You just have to substitute the spaces for hyphens (-). Also, notice that the links work in lowercase too (and I think that the lowercase might even be a requirement in other Git platforms, such as Bitbucket, but I'm not 100% sure about that).